conf/build.manifest

```diff
@@ -1,5 +1,5 @@
 ---
-code_name: "Bluefin"
+code_name: "Cobia"
```

Comment:
@sonicaj I am not sure where this is used, tbh

Reply:
This is used in the manifest we add to update image